internal/command/command_test.go

```diff
@@ -920,12 +920,21 @@ func testCopyDir(t *testing.T, src, dst string) {
 		t.Fatal(err)
 	}
 
-	entries, err := ioutil.ReadDir(src)
+	entries, err := os.ReadDir(src)
 	if err != nil {
 		return
 	}
 
+	var infos []fs.FileInfo
 	for _, entry := range entries {
+		info, err := entry.Info()
+		if err != nil {
+			t.Fatal(err)
+		}
+		infos = append(infos, info)
+	}
```

Comment:
You can avoid this loop to get the `info`s, by simply changing the line in the loop below from `entry.Mode()` to `entry.Type()`

Should work correctly, even though `Type` returns a subset of `FileMode`, since we're only interested in `ModeSymlink` types

Reply:
@RLRabinowitz removing the loop and instead using `Type()` works until the code uses `os.Stat()`, which requires an `fs.FileInfo`.

https://github.com/opentffoundation/opentf/blob/d0655dae23056beeefa7ae0130caac79e3d89e34/internal/command/command_test.go#L948